PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -59,22 +78,7 @@ def applyDeepBtagging( process, postfix="" ) :
        elSource = cms.InputTag('slimmedElectrons'),
        rParam = 0.8,
        jetCorrections = ('AK8PFPuppi', cms.vstring(['L2Relative', 'L3Absolute']), 'None'),
-       btagDiscriminators = [
-          'pfDeepDoubleBJetTags:probQ',
-          'pfDeepDoubleBJetTags:probH',
-          'pfDeepDoubleBvLJetTags:probQCD',
-          'pfDeepDoubleBvLJetTags:probHbb',
-          'pfDeepDoubleCvLJetTags:probQCD',
-          'pfDeepDoubleCvLJetTags:probHcc',
-          'pfDeepDoubleCvBJetTags:probHbb',
-          'pfDeepDoubleCvBJetTags:probHcc',
-          'pfMassIndependentDeepDoubleBvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleBvLJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleCvLJetTags:probHcc',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHcc',
-       ] # + pfDeepBoostedJetTagsAll
+       btagDiscriminators = _btagDiscriminators.names
```

Comment:
Only if you need to update anything else in the PR, please move the comma to the end of this line (as everywhere else) and not at the beginning of the next one

Reply:
fixed